.github/workflows/auto_update_doc.yml

```diff
@@ -16,11 +16,6 @@ jobs:
 
     steps:
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      with:
-        repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
@jcwchen could you help take a look at this PR? Hopefully the changes are fine? Thanks a lot!